Scalable and secure Docker birthday app using Swarm and Load balancer

dvnagesh opened the pull request:
Scalable and secure Docker birthday app using Swarm and Load balancer

conradwt commented:
@dvnagesh I'm not looking at th pull request but it's not 100% clear as to actions taken in the individual commits.

Ch3ck commented:
@dvnagesh It will be better if you squash all your commits into one. and make another PR. It'll be easier to review and possibly merge :santa:

dvnagesh commented:
Hi,

This PR is part of docker birthday app challenge. I saw that we need to submit the entry for app challenge and a PR. Please do let me know if you want me to squash all commits into one and I can do that.  Thanks.

Ch3ck commented:
Please do :+1:

dvnagesh commented:
Hi,

Having some challenges in doing that since I'm new to git commit and push. Anyway, I have submitted another PR with change details which is also given below.

1. Added pre-requisites.md
2. Added Architecture image of scalable birthday app on Swarm at tutorial-images/docker-bday.jpg
3. Updated README.md with Scalable birthday app on Swarm along with link to pre-requisites.
4. Updated tutorial.md with instructions to run Scalable birthday app on Docker swarm.
5. Added voting and worker apps loarbalancer configuration file lb/haproxy.cfg at docker-birthday-3/example-voting-app
6. Updated docker-compose.yml and added loadbalancer service, moved voting/result apps to back-tier and used image for voting/result apps

Apologies for the inconvenience. 

Thanks.
